src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -73,8 +68,8 @@ public Card search(String url) {
       return null;
     }
     // No description or code is allowed for StackOverflow.
-    String description = getDescription(answerBody);
-    String code = getCode(answerBody);
+    String description = getAnswerBody(answerId);
```

Comment:
Oh wait, hmm if the description is null or empty, no card should be created here right?

Reply:
oh wrong method called here, already got the answerbody above